modules/web/src/com/haulmont/sampler/web/ui/components/popupbutton/customlayout/customlayout-popupbutton.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<window xmlns="http://schemas.haulmont.com/cuba/window.xsd"
+        class="com.haulmont.sampler.web.ui.components.popupbutton.customlayout.CustomLayoutPopupButtonFrame"
+        messagesPack="com.haulmont.sampler.web.ui.components.popupbutton.customlayout">
+    <layout>
+        <hbox width="300px">
```

Comment:
Why do we need this? This is an extra layout component which isn't related to the sample's goal.